package com.spring_boot_mybatis.proejct.dao;

import java.util.ArrayList;
import java.util.HashMap;

import com.spring_boot_mybatis.proejct.dto.ProductDTO;


public interface IProductDAO {
	void insertProduct(ProductDTO prdDto);

	void updateProduct(ProductDTO prdDto);

	void deleteProduct(String prdNo);

	ArrayList<ProductDTO> listAllProduct();

	ProductDTO detailViewProduct(String prdNo); // 매개변수와 반환값이 있는 메서드

	String prdNoCheck(String prdNo);

	ArrayList<ProductDTO> productSearch(HashMap<String, Object> map); // 상품검색
}
